lib/common/templates/android/resources.template.js

```diff
@@ -1,50 +1,61 @@
-export default `<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Since this file now imports from `style-dictionary/utils` I was getting errors about "import not allowed outside module" error or whatever. I couldn't find the culprit, I imagine it has something to do with lodash "template" util not allowing ESM imports or something, so I just went ahead and refactored this thing to use native ES6 Template Literals instead, as I'm planning to do with all templates (so we can ditch lodash)